### PR TITLE
[TASK] Migrate form definition

### PR DESCRIPTION
**Pull request by gilbertsoft**
See https://docs.typo3.org/c/typo3/cms-core/master/en-us/Changelog/10.0/Feature-80420-AllowMultipleRecipientsInEmailFinisher.html

### CODE REVIEW

File: Resources/Private/Forms/Contact.form.yaml [hunk 1]
```diff
@@ -8,13 +8,10 @@ finishers:
         identifier: EmailToReceiver
         options:
             subject: '{subject}'
-            recipientAddress: ''
-            recipientName: ''
```
Comment (mbrodala) [inline Resources/Private/Forms/Contact.form.yaml:12]: These have been dropped without replacement?

File: Resources/Private/Forms/Contact.form.yaml [hunk 2]
```diff
@@ -8,13 +8,10 @@ finishers:
...
             senderAddress: '{email}'
             senderName: '{fullname}'
```
Comment (mbrodala) [inline Resources/Private/Forms/Contact.form.yaml:12]: These must also be migrated.

File: Resources/Private/Forms/Contact.form.yaml [hunk 3]
```diff
@@ -8,13 +8,10 @@ finishers:
...
-            replyToAddress: '{email}'
-            carbonCopyAddress: ''
-            blindCarbonCopyAddress: ''
```
Comment (mbrodala) [inline Resources/Private/Forms/Contact.form.yaml:17]: These have been dropped without replacement?

**Review by mbrodala — CHANGES_REQUESTED** [review]

File: Resources/Private/Forms/Contact.form.yaml (hunk 1 above)
Comment (gilbertsoft) [reply Resources/Private/Forms/Contact.form.yaml:12]: Just did the same mentioned in Slack. What should this look like?

```
recipients:
  '': ''
```

File: Resources/Private/Forms/Contact.form.yaml (hunk 2 above)
Comment (gilbertsoft) [reply Resources/Private/Forms/Contact.form.yaml:12]: According the sources this looks still fine https://github.com/TYPO3/typo3/blob/10.4/typo3/sysext/form/Classes/Domain/Finishers/EmailFinisher.php#L54-L55

Comment (mbrodala) [reply Resources/Private/Forms/Contact.form.yaml:12]: Yes, got confused myself. This of course is still a single value.

File: Resources/Private/Forms/Contact.form.yaml (hunk 1 above)
Comment (mbrodala) [reply Resources/Private/Forms/Contact.form.yaml:12]: As mentioned in Slack these options should be skipped in the first place and thus also do not need a migration. Users of the Bootstrap Package will never edit this file to add static values. Instead they will either copy it and add static values or set recipients via finisher overrides.

File: Resources/Private/Forms/Contact.form.yaml (hunk 2 above)
Comment (mbrodala) [reply Resources/Private/Forms/Contact.form.yaml:12]: So these must be left unchanged.

File: Resources/Private/Forms/Contact.form.yaml (hunk 3 above)
Comment (mbrodala) [reply Resources/Private/Forms/Contact.form.yaml:17]: CC and BCC are optional and should be skipped if no useful static value can be set.

**Review by mbrodala — APPROVED** [review]

File: Resources/Private/Forms/Contact.form.yaml (hunk 1 above)
Comment (gilbertsoft) [reply Resources/Private/Forms/Contact.form.yaml:12]: Yes, agree.

**Pull request merged**